website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -89,7 +87,7 @@ An Azure admin will need one of the following permissions in both the Microsoft
 - Azure Service Administrator
 - Azure Co-administrator
 
-If your Azure DevOps account is connected to Entra ID, then you can proceed to [Connect a service user](#connect-a-service-user). However, if you're just getting set up, connect Azure DevOps to the Microsoft Entra ID app you just created:
+If your Azure DevOps account is connected to Entra ID, then you can proceed to [Connect a service principal](#connect-a-service-principal). However, if you're just getting set up, connect Azure DevOps to the Microsoft Entra ID app you just created:
 
 1. From your Azure DevOps account, select **Organization settings** in the bottom left.
 2. Navigate to Microsoft Entra ID.
```

Comment:
This part is incorrect for service principal (unless these are for service user). For service principal, you go to org settings -> navigate to users -> add users -> search and add your application as a service principal in the search bar -> add projects for `Add to projects` and also for `Azure DevOps Groups` select `Project Administrators` (this is important for being able to create webhooks)